index.html

```diff
@@ -233,6 +244,13 @@ <h1>Properties</h1>
 be possible to submit items to the registry without normative definitions (see <a href="#the-registration-process"></a>).
     </p>
 
+<p>
+  Concise Data Defition Lanuage (CDDL) provides a succint data defition for representing the Abstract Data Model of 
+  the <a href="https://www.w3.org/TR/did-core/#core-representations" taret="_blank">DID Core Specification</a> and associated <a href="https://w3c.github.io/did-spec-registries/" target="_blank">DID Spec Registeries</a>.  The CDDL definition for the DID Document Specication and associated 
+  registeries can be found <a href="https://ipfs.io/ipfs/QmNZvYAYqSgUuRwv6AhE3rMDE1nzvTfsq7xrVzXqM2x4Jw" target="_blank">here</a>.  
```

Comment:
It would be better if we checked the representations into the repo so that they live with the DID Spec Registries. We can also point to IPFS locations. The concern with the IPFS locations is that the information may disappear if people stop replicating it. The archival policy at W3C is such that other institutions will take over if W3C fails (e.g., Library of Congress)... none of backup archival providers are staking IPFS hashes right now AFAIK.

Reply:
I agree, feel free to put CDDL in the github repo and link directly to it in github for now.

Reply:
done